tutorials-v4/time-evolution/012_floquet_formalism.md

```diff
@@ -0,0 +1,276 @@
+
+More information on the implementation of the Floquet Formalism in QuTiP can be found in the [documentation](https://qutip.org/docs/latest/guide/dynamics/dynamics-floquet.html).
+
+### Imports 
+
+```python
+import matplotlib.pyplot as plt
+import numpy as np
+from qutip import (about, expect, floquet_markov_mesolve,
+                   floquet_master_equation_rates,
+                   floquet_master_equation_tensor, floquet_modes,
+                   floquet_modes_t, floquet_modes_t_lookup,
+                   floquet_modes_table, floquet_state_decomposition,
+                   floquet_wavefunction, floquet_wavefunction_t, fmmesolve,
+                   ket2dm, num, plot_wigner, rand_ket, sesolve, sigmax, sigmaz)
+```
+
+### System setup
+For consistency with the documentation we consider the driven system with the following Hamiltonian: 
+
+$$ H = - \frac{\Delta}{2} \sigma_x - \frac{\epsilon_0}{2} \sigma_z + \frac{A}{2} \sigma_x sin(\omega t) $$
+
+```python
+# Constants
+delta = 0.2 * 2 * np.pi
+eps0 = 1 * 2 * np.pi
+A = 2.5 * 2 * np.pi
+omega = 1.0 * 2 * np.pi
+T = 2 * np.pi / omega
+
+# Hamiltonian
+H = [
+    -delta / 2.0 * sigmax() - eps0 / 2.0 * sigmaz(),
+    [A / 2 * sigmax(), "sin({w}*t)".format(w=omega)],
+]
+```
+
+### Floquet modes and quasienergies
+For periodic Hamiltonians the solution to the Schrödinger equation can be represented by the Floquet modes $\phi_\alpha(t)$ and the quasienergies $\epsilon_\alpha$. We can obtain these for the initial time $t=0$ by using the function `floquet_modes()`. We can display for example the first floquet mode at $t=0$ using a Wigner distribution.
+
+```python
+f_modes0, f_energies = floquet_modes(H, T)
+plot_wigner(f_modes0[0]);
+```
+
+For the system defined above there are two eigenenergies. For the defined system, we can plot the two quasienergies for varying strength of driving $A$.
+
+```python
+A_list = np.linspace(1.0 * omega, 4.5 * omega, 20)
+quasienergies1, quasienergies2 = [], []
+for A_tmp in A_list:
+    # temporary Hamiltonian
+    H_tmp = [
+        -delta / 2.0 * sigmax() - eps0 / 2.0 * sigmaz(),
+        [A_tmp / 2 * sigmax(), "sin({w}*t)".format(w=omega)],
+    ]
+    # floquet modes and quasienergies
+    _, (e1, e2) = floquet_modes(H_tmp, T, sort=True)
+    quasienergies1.append(e1), quasienergies2.append(e2)
+```
+
+```python
+plt.scatter(A_list / omega, quasienergies1, label="e1")
+plt.scatter(A_list / omega, quasienergies2, label="e2")
+plt.xlabel("A / w"), plt.ylabel("Quasienergies")
+plt.legend();
+```
+
+### Time evolution with floquet mode
+To calculate the time evolution of a random initial state $\psi(0)$, we have to decompose the state in the Floquet basis (formed by the floquet modes).
```

Comment:
super nitpick, sometimes Floquet is capitalized, sometimes not.  Maybe stick with capitalized everywhere?